supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,6 +10,20 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
 
+[[conversational-style]]
+== Conversational style
+
+Follow _The IBM Style Guide_ guidance of _less conversational_ style in most cases. 
+
+As needed, adjust the conversational to _fairly conversational_ for an audience of new users or _least conversational_ for API documentation and other very experienced audiences.
+
+.Example AsciiDoc: Less conversational style 
+
+----
+Get answers now. Search the documentation and posts from the Stack Overflow community, which is actively monitored.
```

Comment:
This isn't really an Asciidoc example, it's a writing example. I would remove the Asciidoc markup and remove Asciidoc from the heading.

We provide Asciidoc examples to demonstrate how to code a specific guideline. That guidance is moot in this example.

(BTW, is it AsciiDoc or Asciidoc?)

Reply:
+1 for removing "AsciiDoc" from the example title, can just be "Example: Less conversational style". And the code block doesn't seem necessary. Could do it like how the linking section does it: https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links

And yeah it is AsciiDoc capital D.

Reply:
According to the asciidoc.org, it's AsciiDoc. Remove AsciiDoc markup.